lib/process-services-cloud/src/lib/form/components/cloud-form-rendering.service.ts

```diff
@@ -1,51 +0,0 @@
-/*!
- * @license
- * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-import { Injectable } from '@angular/core';
-import { FormFieldTypes, FormRenderingService } from '@alfresco/adf-core';
-import { AttachFileCloudWidgetComponent } from './widgets/attach-file/attach-file-cloud-widget.component';
-import { DropdownCloudWidgetComponent } from './widgets/dropdown/dropdown-cloud.widget';
-import { DateCloudWidgetComponent } from './widgets/date/date-cloud.widget';
-import { PeopleCloudWidgetComponent } from './widgets/people/people-cloud.widget';
-import { GroupCloudWidgetComponent } from './widgets/group/group-cloud.widget';
-import { PropertiesViewerWidgetComponent } from './widgets/properties-viewer/properties-viewer.widget';
-import { RadioButtonsCloudWidgetComponent } from './widgets/radio-buttons/radio-buttons-cloud.widget';
-import { FileViewerWidgetComponent } from './widgets/file-viewer/file-viewer.widget';
-import { DisplayRichTextWidgetComponent } from './widgets/display-rich-text/display-rich-text.widget';
-import { DataTableWidgetComponent } from './widgets/data-table/data-table.widget';
-
-@Injectable({
-    providedIn: 'root'
-})
-export class CloudFormRenderingService extends FormRenderingService {
```

Comment:
isn't this a breaking change?

Reply:
I have just moved from one folder to another that is more meaningful, but inside the same module. If you think it can break anything, I can put it back to were it was

Reply:
I adopted the proposed solution from @BSekula for creating a dedicated service for the display mode service, so this file remains now unchanged